tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -24,10 +24,10 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
   var validDocumentId: DocumentId = null
```

Comment:
Do you think it's worth adding any additional unit tests to verify the new doc create option we're adding here?

Reply:
Unsure. It won't compile/work without it being there, but there isn't any functionality involved here. It would blow up if the option wasn't going all the way through. There are tests in `ironoxide` to actually test it.

I certainly could have a test that encrypts to only a group and shows that happened. I'm not sure if it's testing something valuable about the Scala SDK code though.

Reply:
>  I'm not sure if it's testing something valuable about the Java SDK code though.

I'm pretty sure you could have made that argument about 75% of the tests that we have in this repo already. In my opinion I'd just add another test just to follow the convention of the tests in this repo doing full end-to-end tests that test much more than just the binding code.